src/config.rs

```diff
@@ -1016,10 +1036,12 @@ impl Config {
     ///
     /// Uses default values for everything except for the cache and TAL
```

Comment:
With the `tal_dir` argument gone, this comment is outdated. Moreover, the function could (should?) be renamed to singular form.